### PR TITLE
Hw node express

### PR DESCRIPTION
**Pull request by Yevhenii-Bezeha**

### CODE REVIEW

**Comment by Yevhenii-Bezeha** [comment]
I used ts in order to practice in node environment. Also, I did CRUD for POST entity in mongodb, because we already learned it, so can use it.
And I'd like to mention the next commit is not connected with backend part, I changed the angular structure after angular routes lecture. It was easier to do it in this PR.

File: .env [hunk 1]
```diff
@@ -0,0 +1,2 @@
+PORT=3000
+DB_HOST=mongodb+srv://test:test@cluster0.vus38.mongodb.net/test
```
Comment (afandul) [inline .env:2]: Do not include env file to SCM

File: backend/bin/index.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
+import app from '../src/app';
```
Comment (afandul) [inline backend/bin/index.ts:1]: index should be rather in 'src' than in 'bin'. This is a source file, not an .exe/.sh one

File: backend/src/app.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
+"use strict";
```
Comment (afandul) [inline backend/src/app.js:1]: Do not include built files (JS) to SCM

File: backend/src/models/IPost.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
+export interface IPost {
```
Comment (afandul) [inline backend/src/models/IPost.ts:1]: Do not prefix interfaces with 'I':
 https://github.com/microsoft/TypeScript-Handbook/issues/121

**Review by afandul — COMMENTED** [review]
Nice solution with TS. Pay attention to comments

**Pull request merged**